versions.props

```diff
@@ -5,7 +5,7 @@ org.apache.flink:* = 1.14.3
 org.apache.hadoop:* = 2.7.3
 org.apache.hive:* = 2.3.8
 org.apache.httpcomponents.client5:* = 5.1
-org.apache.orc:* = 1.7.3
+org.apache.orc:* = 1.7.4
```

Comment:
Would you mind to remove this `TODO` also ? 

https://github.com/apache/iceberg/pull/3784/files#diff-e7fcc622bb5551f5158e35bd0e929e6eeec73717d1a01465eaa691ed098af3c0R95